.github/workflows/cicd-stg.yml

```diff
@@ -0,0 +1,196 @@
+name: "CICD staging"
+
+on:
+  # Test run before merging
+  pull_request:
+    branches:
+      - main
+      - users/richard/readd-cicd
+  # On merged
+  push:
+    branches:
+      - main
+
+jobs:
+  build-docker:
+    permissions:
+      id-token: write
+      contents: write
+    runs-on: ubuntu-22.04
+    if: github.actor != 'dependabot[bot]' && github.ref_name == 'main'
+    name: "Deploy image"
+    strategy:
+      fail-fast: false
+      matrix:
+        application:
+          [
+            devex-apollo,
+            dev-wallet,
+            faucet-service,
+            governance-api,
+            governance-snapshot,
+            multisig,
+            neo-savant,
+            scilla-server,
+            zillion,
+            zilliqa-bridge-validator,
+            zilliqa-bridge-web,
+          ]
+        include:
+          - application: devex-apollo
```

Comment:
I am not sure we want to deploy all these app images :)

Reply:
Good point well made - sorry! - removed.